### ISSUE TITLE
[Question] Moonshot compatibility with forward proxy server

### ISSUE THREAD

**Issue by SidPad03** [aiverify-foundation/moonshot#301]
Hi all,  

Our company is working on a project that requires capturing and analyzing traffic data exchanged between our client application, in this case, Moonshot, and our host server via a forward proxy server configured to monitor and manage the traffic. The forward proxy server will act as a firewall, so all client-host traffic must be mediated by this forward proxy server. 

Could someone advise me on the necessary modifications/setup required for Moonshot to ensure it is compatible with this arrangement?

**Comment by saddepalli** [comment]
Yes, this would be a great feature as many organizations have explicit proxies enabled.

In case of browsers, proxy settings can be made via PAC file.
In case of curl as a client, one can use command like this: curl -x "http://user:pwd@<proxy ip>:<proxport>" "chatgpt.com"

To test public LLMs using Moonshot project via proxies would be a great addition.

**Comment by imda-tzx** [comment]
Thank you for these feedback. Unfortunately we currently don't have plans to support this
